curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/understanding-boolean-values.md

```diff
@@ -9,24 +9,24 @@ dashedName: understanding-boolean-values
 
 # --description--
 
-Another data type is the <dfn>Boolean</dfn>. `Booleans` may only be one of two values: `true` or `false`. They are basically little on-off switches, where `true` is "on" and `false` is "off." These two states are mutually exclusive.
+Another data type is the <dfn>Boolean</dfn>. Booleans may only be one of two values: `true` or `false`. They are basically little on-off switches, where `true` is "on" and `false` is "off". These two states are mutually exclusive.
 
 **Note**  
-`Boolean` values are never written with quotes. The `strings` `"true"` and `"false"` are not `Boolean` and have no special meaning in JavaScript.
+`boolean` values are never written with quotes. The strings `"true"` and `"false"` are not `boolean` and have no special meaning in JavaScript.
```

Comment:
@nhcarrigan I am thinking the word Boolean should be translated to whatever it is in each language.  Also, this is one of the only times I would accept the double quotes inside the backticks.
```suggestion
Boolean values are never written with quotes. The strings `"true"` and `"false"` are not Boolean and have no special meaning in JavaScript.
```

Reply:
Boolean should be translated, yes (does it translate? Isn't it a name?).

I don't feel that the quotes are necessary there, because we specifically say "strings".